samples/apps/spring-petclinic/db-deployment.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: spring-petclinic-postgresql
+stringData:
+  database: petclinic
+  username: petclinic
```

Comment:
since these values are same as those set in `application-postgres.properties` file, how do we know that they are really consumed from binding projection?

Reply:
good point, I've change changed it to someting different to be sure